src/calibrate_geds.jl

```diff
@@ -12,7 +12,9 @@ Also calculates the configured cut/flag values.
 """
 function calibrate_ged_channel_data(data::LegendData, sel::AnyValiditySelection, detector::DetectorIdLike, channel_data::AbstractVector; 
         e_cal_pars_type::Symbol=:rpars, e_cal_pars_cat::Symbol=:ecal,
-        psd_cal_pars_type::Symbol=:ppars, psd_cal_pars_cat::Symbol=:aoe, psd_cut_pars_type::Symbol=:ppars, psd_cut_pars_cat::Symbol=:aoe,
+        psd_pars_type::Symbol=:ppars,
+        aoe_cal_pars_type::Symbol=:ppars, aoe_cal_pars_cat::Symbol=:aoe, aoe_cut_pars_type::Symbol=:ppars, aoe_cut_pars_cat::Symbol=:aoe,
```

Comment:
If `cut` and `cal` are always identical, you could do:
```suggestion
       aoe_pars_type::Symbol=:ppars, aoe_pars_cat::Symbol=:aoe,
```

Reply:
They are actually not. There is still the option to use run-based calibrations with partition wide cuts. So this has to stay.